crossdock/main_test.go

```diff
@@ -95,6 +95,16 @@ func TestCrossdock(t *testing.T) {
 				"transport": []string{"http", "tchannel"},
 			},
 		},
+		{
+			name: "ctxpropagation",
+			axes: axes{
+				"transport": []string{"http", "tchannel"},
```

Comment:
I couldn't tell from reading the test - do we every criss-cross transports in the same test? I want to make sure we aren't breaking the chain...